modules/swagger-codegen/src/main/resources/csharp/ApiClient.mustache

```diff
@@ -22,15 +22,16 @@ using RestSharp.Portable.HttpClient;
 using RestSharp;
 {{/netStandard}}
 
-namespace {{packageName}}.Client
-{
+namespace {{packageName}}.Client {
     /// <summary>
     /// API client is mainly responsible for making the HTTP call to the API backend.
     /// </summary>
-    {{>visibility}} partial class ApiClient
-    {
-        private JsonSerializerSettings serializerSettings = new JsonSerializerSettings
-        {
+    {{>visibility}} partial class ApiClient {
+
+        private static Logger log = new Logger(nameof(ApiClient));
```

Comment:
`nameof` is not compatible with all supported .NET versions here. This should just hardcode the `ApiClient` string as that will never change, and if it does it's a breaking change.